### PR TITLE
statCalc testing coverage

### PR DESCRIPTION
**Pull request by rimangan**

### CODE REVIEW

File: cmd/statCalc/statCalc.go [hunk 1]
```diff
@@ -6,202 +6,148 @@ import (
...
-	log.SetFlags(log.Ldate | log.Ltime | log.Lshortfile)
-	flag.Parse()
-
-	MultipleOptionErrorCheck(Normal, Binomial, Poisson, Beta, Gamma, SampleAfs, SampleBeta, SampleGamma, SampleNormal)
-	common.RngSeed(*randSeed, *setSeed)
-	if *Normal != "" {
-		words := strings.Split(*Normal, ",")
+func statCalc(s Settings) {
+	MultipleOptionErrorCheck(s.Normal, s.Binomial, s.Poisson, s.Beta, s.Gamma, s.SampleAfs, s.SampleBeta, s.SampleGamma, s.SampleNormal)
+	common.RngSeed(s.RandSeed, s.SetSeed)
```
Comment (craiglowe) [inline cmd/statCalc/statCalc.go:19]: Did we come to a conclusion on the idea of removing random seeding in most places?

**Review by craiglowe — APPROVED** [review]
Great!

File: cmd/statCalc/statCalc.go (hunk 1 above)
Comment (rimangan) [reply cmd/statCalc/statCalc.go:19]: yes but that's a major change. Should be its own request

**Review by cshoben — APPROVED** [review]

**Review by RavenYL — APPROVED** [review]

**Review by craiglowe — APPROVED** [review]
Great to have these tests!

**Pull request merged**